igamt-lite-controller/src/main/java/gov/nist/healthcare/tools/hl7/v2/igamt/lite/web/controller/NotificationController.java

```diff
@@ -27,9 +62,44 @@ public Notifications findNotificationByIgId(@PathVariable("id") String id)
     return notificationsRepository.findByIgDocumentId(id);
   }
 
-
-  @RequestMapping(value = "/delete/{id}", method = RequestMethod.POST, produces = "application/json")
+  @RequestMapping(value = "/delete/{id}", method = RequestMethod.POST,
+      produces = "application/json")
   public void deleteNotification(@PathVariable("id") String id) throws DataNotFoundException {
     notificationsRepository.delete(id);
   }
+
+  @RequestMapping(value = "/phinvadsEmailNotification/{id}", method = RequestMethod.GET,
```

Comment:
name it for what it is. this is to subscribe so maybe /notifications/{id}/send/{accountId} , where id is the notification and accountId is the id of the account. 
You can't rely on the session to find the user since there is none available when the thread is doing its work.  
Sorry Woo. We should have agreed on the arguments first before you implement. But it is ok. 

What do you think ?